app/controllers/openid_connect/authorization_controller.rb

```diff
@@ -32,7 +32,7 @@ def check_sp_active
     end
 
     def check_sp_handoff_bounced
-      return unless SpHandoffBounce.is_bounced?(session)
+      return unless SpHandoffBounce::IsBounced.call(sp_session)
```

Comment:
I thought we had already reverted this in our last patch deploy: https://github.com/18F/identity-idp/commit/78680fc1e0659ff56d88a704abf9c526d61afed1

Reply:
this is odd, but https://github.com/18F/identity-idp/blob/stages/prod/app/controllers/openid_connect/authorization_controller.rb#L34-L35 does have the change already

Reply:
it's probably fine? the result is still the desired result I think

Reply:
Did that not get merged back into `master`?